src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -1,4 +1,157 @@
 package seedu.duke.parser;
 
+import seedu.duke.command.*;
+import seedu.duke.exception.DukeException;
+
 public class Parser {
+    public static final String COMMAND_ADD = "add";
+    public static final String TASK_KEYWORD = "task";
+    public static final String LESSON_KEYWORD = "lesson";
+    public static final String ALL_KEYWORD = "all";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_DONE = "done";
+    public static final String COMMAND_DELETE = "delete";
+    public static final String COMMAND_FIND = "find";
+    public static final String FLAG_DAY = "-d";
+    public static final String FLAG_INFORMATION = "-i";
+    public static final String FLAG_START_TIME = "-s";
+    public static final String FLAG_END_TIME = "-e";

```

Comment:
Consider refactoring these to `enum`, to reduce clutter on the `Parser` class. But we can do this at another time, doesn't have to be this PR.